edb.desktop

```diff
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=edb
-GenericName=edb debugger
-Comment=edb debugger 
+GenericName="Evan's Debugger"
+Comment="edb is a cross platform x86/x86-64 debugger"
```

Comment:
Actually, ARM port is in progress. Namely, EDB already builds on AArch32 and even lets you do something. So it's not limited to x86/x86-64.

Reply:
If the "x86" part is your only concern, I can omit it :)

Reply:
To be fair, the README and project description on github still say "x86/x86_64"... which I will address now :-).